src/algorithms/Algorithm_Brute_Force.py

```diff
@@ -0,0 +1,128 @@
+import random
+
+
+class AlgorithmBruteForce(object):
```

Comment:
be consistent on the naming of your algorithms, with the algorithm word as prefix or just directly the name for all the algorithms